migrate_group

```diff
@@ -0,0 +1,54 @@
+#! /bin/bash
+
+# Exit on error
+set -e
+# Turn off warning that the site is in GIT mode
+export TERMINUS_HIDE_GIT_MODE_WARNING=1
+
+# Skip the custom code to sync group content. See portland_groups.module
```

Comment:
Do you think should we enable maintenance mode during the migration so users don't see any groups in a half-baked state? Or would that be too much downtime for users and we should just take the risk? (maybe you've already thought about this)

Reply:
Good idea! Added the drush command.